packages/mapper/src/Context.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Tempest\Mapper;
+
+/**
+ * Provides context information to casters, serializers and mappers.
+ */
+interface Context
```

Comment:
I know it's more verbose, but `MapperContext` sounds more clear to me. `Context` is rather vague.